recipes/yattag/meta.yaml

```
@@ -0,0 +1,37 @@
+{% set name = "yattag" %}
+{% set version = "1.11.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 94210945c868f696a31812a510c01724d61c9a4d279eb4adf340c0d2a4c0bfd7
+
+build:
+  number: 0
+  noarch: python
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - yattag
+
+about:
+  home: http://www.yattag.org
+  license: GNU Library or Lesser General Public (LGPL)
```

From https://opensource.org/licenses/alphabetical:
```suggestion
  license: LGPL-2.1
```